nimble/controller/include/controller/ble_ll.h

```diff
@@ -34,6 +35,14 @@ extern "C" {
 #error 32.768kHz clock required
 #endif
 
+#define BLE_LL_ASSERT(cond) \
+    if (!(cond)) { \
+        ble_ll_hci_ev_send_vendor_err(__FILE__, __LINE__); \
+        if (hal_debugger_connected()) { \
+            assert(0);\
```

Comment:
bkpt instead of assert

Reply:
well,  actually I will remove hal_debugger_connected - each architecture will do on assert what  is needed.

Reply:
perhaps cleaner way to do this is to send an event, flush transport or wait for everything to be sent to host (may need some update in transport) and then just assert. or just send and do infinite loop - calling hal is not really needed here. or simply add `#if MYNEWT` around hal call.